specification/dimensions/pricingmodel.md

```diff
@@ -0,0 +1,38 @@
+# Pricing Model
+
+Pricing Model indicates the type of unit price applied to a charge at the time it was used or purchased. Pricing Model is commonly used to differentiate between charges at on-demand vs. reduced prices when investigating optimization opportunities, like for commitment-based discount coverage.
+
+The PricingModel column MUST be present and MUST NOT be null or empty. This column is of type String and MUST be one of the allowed values. PricingModel MUST be "Commitment-Based Discount" when CommitmentDiscountId is not null. PricingModel MUST be "Dynamic" when pricing is determined by the provider and may change over time, regardless of predetermined agreement pricing.
```

Comment:
Capitalise not for CommitmentDiscountId is not null

Reply:
Looking at the merged items in our `working_draft`, there's no need to capitalize when it's inside the condition part.

[Statement] if [condition], or
[Statement] when [condition].